packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -0,0 +1,11 @@
+$p-1: 4px !important;
```

Comment:
Added a global stylesheet as well as separate stylesheets for templates and partials